rpm/SPECS/openresty-openssl.spec

```diff
@@ -35,26 +39,22 @@ Provides C header and static library for OpenResty's OpenSSL library.
 
 %prep
 %setup -q -n openssl-%{version}
-
 %patch0 -p1
 
 
 %build
-./config --prefix=%{openssl_prefix} \
-    no-threads \
-    shared -g
-
+./config --openssldir=%{_prefix} no-threads shared zlib -g \
+-I%{openresty_prefix}/include \
+-L%{openresty_prefix}/%{_lib} \
+-Wl,-rpath,%{openresty_prefix}/%{_lib}
```

Comment:
The zlib headers and libs should be installed into the prefix `%{openresty_prefix}/zlib/`.